os/arch/arm/src/armv7-m/up_memfault.c

```diff
@@ -69,13 +74,12 @@
  * Pre-processor Definitions
  ****************************************************************************/
 
-#undef DEBUG_MEMFAULTS			/* Define to debug memory management faults */
-
-#ifdef DEBUG_MEMFAULTS
-#define mfdbg(format, ...) lldbg(format, ##__VA_ARGS__)
-#else
-#define mfdbg(...)
-#endif
+#define	MMARVALID	0x00000080
```

Comment:
Pls remove multiple spaces between #define and MMxxxx

Reply:
Done.